src/runtime/policy/policy.ts

```diff
@@ -232,6 +233,16 @@ export class PolicyTarget {
     });
   }
 
+  getRestrictedFields() {
```

Comment:
Some documentation about what a restricted field is (and same for restricted type) would be great.